src/main/java/com/ncc/aif/ValidateAIF.java

```diff
@@ -154,13 +154,13 @@ public static ValidateAIF create(ImmutableSet<CharSource> domainOntologySources,
     // Show usage information.
```

Comment:
Y'know, I did a quick search for something to help with parsing args, but I didn't come across picocli unfortunately.  The two solutions I found were over-engineered and involved adding another large-ish third-party library dependency just for one CLI...didn't seem worth it.  Picocli seems like it might be more in-line with our needs.  I'll write a JIRA ticket to examine it for use in the validator.  Feel free to edit/comment it after it's created with your thoughts.

Reply:
Wrote AIDA-696 to capture this.